raiden/tests/scenarios/README.md

```diff
@@ -0,0 +1,55 @@
+## Scenarios
+
+The scenarios listed within this folder are getting executed nightly by the [scenario player](https://github.com/raiden-network/scenario-player/) or can be executed locally after installing the scenario player.
+
+## Purpose
+
+The scenarios serve the following purpose
+
+- Full End-to-End Tests covering contracts, transport, Raiden client, blockchain, eth-client, monitoring service, pathfinding service and service contracts
+- Testing on a "dirty" blockchain
+- User Acceptance tests
+- Testing in a distributed environment
+- Understanding how Raiden and the Raiden Services can be used and interact
+
+## Scenarios
+
+#### [ms1_simple_monitoring](./ms1_simple_monitoring.yaml)
+
+A channel between two nodes is opened, a transfer is made. Then, node 1 goes offline and node 0 closes the channel. The last assert checks whether the monitoring service interferes and received its reward.
+
+#### [ms2_simple_monitoring](./ms2_simple_monitoring.yaml)
+
+A channel between two nodes is opened, a transfer is made. Then, node 1 goes offline and node 0 closes the channel. After the monitoring trigger block is passed node1 gets back online.
```

Comment:
maybe add, that "node 1 does not get back online in time and cannot update the channel info by herself." the monitoring services interferes and gets its reward